src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.cs

```diff
@@ -626,7 +631,13 @@ public ReanalyzeScope(IEnumerable<ProjectId> projectIds = null, IEnumerable<Docu
 
             public string GetLanguages(Solution solution)
```

Comment:
>GetLanguages [](start = 26, length = 12)

Perhaps rename to GetLanguageList since this doesn't return IEnumerable<string>, so it's different than GetDocuments below.